### DISCUSSION TITLE
How to configure camera?

### DISCUSSION THREAD

**Discussion by garyturell** [vlachoudis/bCNC#1666] [Q&A]
I am using bcnc and see in the wiki that i am supposed to Select the camera from "Tools -> Config -> Camera".
Where is the Tools button or menu?

**Comment by MARIOBASZ** [comment]
rename tool as probe

**Reply by garyturell** [reply]
Mario, Thanks for replying. I am not sure what that means. I don't know how to rename a button and when I try the button, there is no Config option to select a camera. 
I am trying to select the correct camera to use with bcnc, I have two cameras and the default one is not the one I want to use. I am looking for a way to choose the camera.
